### PR TITLE
Guava 21.0 -> 24.0

### PR DESCRIPTION
**Pull request by sullis**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -75,7 +75,7 @@
         <airline.version>2.2.0</airline.version>
         <antlr.version>4.7</antlr.version>
         <csv.version>1.4</csv.version>
-        <guava.version>21.0</guava.version>
+        <guava.version>24.0-jre</guava.version>
```
Comment (apurvam) [inline pom.xml]: what is this `-jre` suffix?

Comment (sullis) [reply pom.xml]: Guava 24.0 has two published releases:  "-android" for the Android platform.  "-jre" for Oracle Java 8 and above.

Comment (sullis) [reply pom.xml]: see:  https://github.com/google/guava

**Comment by sullis** [comment]
💯

**Pull request merged**